documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -770,80 +697,48 @@ where:
 
    Like ``line_proj``, ``sphere_proj`` can be :ref:`defined using
    positional or keyword arguments <poskwarg>`. However the same
-   recommendation applies that positionals should only be used to
+   recommendation applies that positional should only be used to
    define ``u`` and ``v``.
 
-``sphere_proj`` defines a spherical coordinate system, where:
-
-* |Q|
-
-  The radius from the origin (``offset``) in :math:`hkl`
-
-* :math:`\theta`
-
-  The angle measured from :math:`\vec{u}` to the vector (:math:`\vec{q}`),
-  which means a :math:`\theta` value of :math:`0^{\circ}` means that :math:`\vec{q}` is parallel 
-  to :math:`\vec{u}`; and a :math:`\theta` value of :math:`90^{\circ}` means that 
-  :math:`\vec{q}` is perpendicular to :math:`\vec{u}`.
-
-  Mathematically this is defined as:
-
-  .. math::
-
-     \cos\left(\theta{}\right) = \frac{\vec{q}\cdot\vec{u}}{\left|q\right|\cdot\left|u\right|}
-
-* :math:`\phi`
-
-  The angle measured between the :math:`u`-:math:`v` plane to the vector
-  (:math:`\vec{q}`), i.e. :math:`0^{\circ}` means that :math:`\vec{q}` lies in the :math:`u`-:math:`v`
-  plane and :math:`90^{\circ}` means that :math:`\vec{q}` is normal 
-  to the :math:`u`-:math:`v` plane
-  (i.e. parallel to :math:`w`).
+``sphere_proj`` defines a spherical coordinate system which contains 
+spherical coordinates of momentum transfer vector  :math:`\vec{Q}`.
+Energy transfer coordinate for ``sphere_proj`` remain unchanged. 
+If projection ``offset`` parameter is zero, this vector is the vector
+of momentum transfer from neutron to excitations -- lattice measured
+in scattering experiment. If offset is not zero, :math:`\vec{Q}`
+is the difference between ``offset`` and the measured momentum transfer.
```

Comment:
After discussion with Toby about incorrect formulas all formulas have been removed. 
The section about orthogonal system - base for spherical coordinate system has been added